.env.example

```diff
@@ -1,2 +1,4 @@
-DATABASE_URL=mysql://root:default_password@localhost:3306/checkpoint
+MYSQL_ROOT_PASSWORD=default_password
+MYSQL_DATABASE=checkpoint
 
+DATABASE_URL=mysql://root:${MYSQL_ROOT_PASSWORD}@mysql:3306/${MYSQL_DATABASE}
```

Comment:
Unfortunately, this concatenation wouldn't work as expected

Reply:
ok I change